### PR TITLE
(PE-30693) change project_ref to versioned_project

### PR DESCRIPTION
**Pull request by steveax**
This commit changes the project_ref parameter to versioned_project
to match the parameter in puppetserver.

!no-release-note

### CODE REVIEW

File: lib/bolt_server/transport_app.rb [hunk 1]
```diff
@@ -514,10 +514,10 @@ def make_winrm_target(target_hash)
 
     # Fetches the metadata for a single plan
     #
-    # @param project_ref [String] the project to fetch the plan from
+    # @param versioned_project [String] the project to fetch the plan from
     get '/project_plans/:module_name/:plan_name' do
-      return MISSING_PROJECT_REF_RESPONSE if params['project_ref'].nil?
-      in_bolt_project(params['project_ref']) do |context|
+      return MISSING_PROJECT_REF_RESPONSE if params['versioned_project'].nil?
```
Comment (McdonaldSeanp) [inline lib/bolt_server/transport_app.rb:519]: Nit: we could probably change this to `MISSING_VERSIONED_PROJECT_RESPONSE` to be a little clearer

But definitely don't block on that :)

**Review by McdonaldSeanp — APPROVED** [review]
👍

File: lib/bolt_server/transport_app.rb (hunk 1 above)
Comment (steveax) [reply lib/bolt_server/transport_app.rb:519]: Yep, good call, let me change that

Comment (steveax) [reply lib/bolt_server/transport_app.rb]: Updated

**Review by donoghuc — APPROVED** [review]

**Pull request merged**